hk-package-clean-except/action.yml

```diff
@@ -19,6 +19,11 @@ inputs:
     required: false
     default: 'pyansys'
     type: string
+  allow-last-days:
+    description: "Avoid removing the last N days images: e.g. '2'."
+    required: false
+    default: ''
+    type: string

```

Comment:
Is there no 'int' type here?

Reply:
Sadly, no... I wish there were. I think the types allowed are these only https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputsinput_idtype

Even if it is not for the same location, that's the types they accept. In fact, everything is a string behind the scenes...